apps/drain-safe/src/components/Icon.tsx

```diff
@@ -12,9 +12,11 @@ const IconImg = styled.img`
   width: auto;
 `;
 
-function Icon(props: Props): JSX.Element {
+function Icon(props: Props): JSX.Element | null {
+  const [noIcon, setNoIcon] = useState<boolean>(false);
   const { logoUri, symbol } = props;
-  return <IconImg src={logoUri} alt={symbol} />;
+
+  return noIcon ? null : <IconImg src={logoUri} alt={symbol} onError={() => setNoIcon(true)} />;
```

Comment:
Would it make sense to use the 'assets' icon from safe-react-components inside a greyed circle, similar as we do in the assets tab?
https://components.gnosis-safe.io/?path=/docs/data-display-icon--icons

![image](https://user-images.githubusercontent.com/25051234/116974504-84ff0000-acbe-11eb-90ca-40a59680f65e.png)

![image](https://user-images.githubusercontent.com/25051234/116974545-921bef00-acbe-11eb-8205-789a48987318.png)

Reply:
The assets icon is used in the sidebar menu so I think it might be confusing.
I've added a question mark icon as a fallback (like Uniswap).

<img width="156" alt="Screenshot 2021-05-05 at 09 24 26" src="https://user-images.githubusercontent.com/381895/117108608-bbeb1980-ad83-11eb-9039-2afb0ad6441d.png">